JoyboyCommunity/assets/svgs/svgComponents/DoubleCopy.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import Svg, {SvgProps, Path} from 'react-native-svg';
+const DoubleCopyIcon = (props: SvgProps) => (
+  <Svg width={24} height={24} fill="none" {...props} viewBox="0 0 24 24">
```

Comment:
Icons moved to the [icons.tsx](https://github.com/keep-starknet-strange/joyboy/blob/be01de1509e512a128715d04cfc5b90beee49fc1/JoyboyCommunity/src/assets/icons.tsx) file, can you move the icons to there?